README.markdown

```diff
@@ -1253,65 +1705,91 @@ $value = try_get_value($data, 'a/b/c/d', 'not_found')
 # using custom separator
 $value = try_get_value($data, 'a|b', [], '|')
 # $value = ['b1','b2','b3']
-~~~
+```
 
 1. **$data** The data structure we are working with.
 2. **'a/b/2'** The path string.
 3. **'not_found'** The default value. It will be returned if nothing is found.
-   (optional, defaults to *undef*)
+   (optional, defaults to *`undef`*)
 4. **'/'** The path separator character.
    (optional, defaults to *'/'*)
 
-#### `type3x`
+*Type*: rvalue.
+
+#### `type3x` TODO: is this deprecated at this point?
```

Comment:
It claims to be deprecated in the docs however there are no warnings which would make me worry about removing it. Most likely it will stay in and we will add a deprecation warning, then in the release after it will be fully removed. 
Arguments can be any of the following:

* string
* array
* hash
* float
* integer
* boolean